src/screens/Signup/index.tsx

```diff
@@ -33,6 +34,9 @@ function Signup() {
 
   const [, loading, error, sendRequest] = useLazyRequest({
     request: signUp,
+    withPostSuccess: () => {
+      history.push('/');
```

Comment:
It may be a good idea to have constants defined for the app paths. So the code is more descriptive
```suggestion
      history.push(HOME_PATH);
```